tests/app/questionnaire/test_router.py

```diff
@@ -0,0 +1,248 @@
+from unittest.mock import MagicMock
+
+from tests.app.app_context_test_case import AppContextTestCase
+from app.questionnaire.router import Router
+from app.questionnaire.location import Location
+from app.questionnaire.completeness import Completeness
+from app.utilities.schema import load_schema_from_params
+
+
+class TestRouter(AppContextTestCase):
```

Comment:
There's no need to inherit from `AppContextTestCase` here. It'll set up the app and DynamoDB for each test but since these tests don't require that you could simply inherit from `TestCase`